package pk;

class test {
	void func() {
		try {
		    throw new Exception();
		} catch (Exception e) {

		} catch (Exception e) {

		}

	}
}
